compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/DecomposeMapScatter.cpp

```diff
@@ -355,9 +356,10 @@ static LogicalResult decomposeToScatter(MapScatterOp mapScatterOp,
 
   SmallVector<Value> offsets = {
       arith::ConstantIndexOp::create(rewriter, loc, 0)};
-  rewriter.replaceOpWithNewOp<vector::ScatterOp>(mapScatterOp, flatOutputBuffer,
-                                                 offsets, indexVector,
-                                                 maskVector, inputVector);
+  SmallVector<Value> operands = {flatOutputBuffer, offsets[0], indexVector,
+                                 maskVector, inputVector};
+  rewriter.replaceOpWithNewOp<vector::ScatterOp>(mapScatterOp, TypeRange{},
+                                                 operands);
```

Comment:
This change might be related to https://github.com/llvm/llvm-project/commit/7e7ea9c5357efcdf9ba6bd7ea3669e607a9af400 (which I authored).
To better follow the coding standards (https://llvm.org/docs/CodingStandards.html#comment-formatting
), I suggest using the following instead:

```suggestion
  rewriter.replaceOpWithNewOp<vector::ScatterOp>(mapScatterOp, /*resultTypes=*/TypeRange{},
                                                 operands);
```